api/src/constants/constants.ts

```diff
@@ -31,10 +31,16 @@ export const TEMPLATE_CONFIG: Record<
   }
 > = {
   JavaScript: {
+    // node binary name does not change from one version to another
     binary: '/usr/local/bin/node',
+    // for scone 5.7 this was necessary
     sconeCuratedImage:
       'registry.scontain.com:5050/sconecuratedimages/node:14.4.0-alpine3.11',
   },
+  'Python3.13': {
```

Comment:
Introduced "Python3.13" templates (differs from "Python" templates due to the binary location)